extended/src/main/java/apoc/vectordb/ChromaDb.java

```diff
@@ -129,7 +129,7 @@ public Stream<EmbeddingResult> get(@Name("hostOrKey") String hostOrKey,
                                                       @Name("collection") String collection,
                                                       @Name("ids") List<Object> ids,
                                                       @Name(value = "configuration", defaultValue = "{}") Map<String, Object> configuration) throws Exception {
-        return getCommon(hostOrKey, collection, ids, configuration, true);
+        return getCommon(hostOrKey, collection, ids, configuration, false);
```

Comment:
in general it would be good to avoid boolean variables, rather use an enum, it's confusing esp. if it changes meaning during development.
and what does "updateMode" mean :) 
don't we also have 3 modes? READ_ONLY, UPDATE_EXISTING, CREATE_IF_MISSING?